src/aggregate/aggregate_exec.c

```diff
@@ -345,7 +345,7 @@ void sendChunk(AREQ *req, RedisModule_Reply *reply, size_t limit) {
 
     RedisModule_ReplyKV_Array(reply, "error"); // >errors
       if (rc == RS_RESULT_TIMEDOUT) {
-        RedisModule_Reply_SimpleString(reply, "Timeout limit was reached");
+        RedisModule_Reply_Error(reply, QueryError_Strerror(QUERY_TIMEDOUT));
```

Comment:
Is this line covered this line by the tests?
I see the `test_resp3:test_search_timeout` but codecov reported this line as uncovered.

Reply:
The test in `test_resp3:test_search_timeout` is failing before, in `prepareExecutionPlan` (timed out there).
I'll add a test failing here.